src/cli/setup.ts

```diff
@@ -4,7 +4,12 @@ import { parseConfigYaml, setupStarbase } from '../starbase';
 export function setup() {
   return createCommand('setup')
     .description('bootstraps or updates graph integration projects')
+    .option(
+      '-c, --config <path>',
+      'optional path to config file',
+      'config.yaml',
+    )
     .action(async (options) => {
-      await setupStarbase(await parseConfigYaml('config.yaml'));
+      await setupStarbase(await parseConfigYaml(options.config));
```

Comment:
The setup command wasn't following the -c flag, but it should have.  Adding as a bonus fix.